### ISSUE TITLE
[SALTSTACK] 

### ISSUE THREAD

**Issue by moondog69** [sans-dfir/sift#621]
SALTSTACK is great but there is just one issue. When it doesn't work there is no documentation on what to do. I installed the latest Jammy and ran the installer all but 32 states installed correctly. This is the message I get

"WARN[0575] first failed state                            comment="One or more requisite failed: sift.python-packages.volatility.sift-python-volatility-community-plugins" component=installer run_num=282 sls=sift.python-packages.volatility"
INFO[0575] statistics                                    component=installer failed=32 success=643 total=675
INFO[0575] salt-call completed but had failed states     component=installer
FATA[0575] salt-call completed but had failed states    
[castsiftinstall.txt](https://github.com/teamdfir/sift/files/14296619/castsiftinstall.txt)

**Comment by ekristen** [comment]
Appreciate the feedback. We can definitely look to improve the documentation around this. 

```
INFO[0670] log file location                             component=installer file=/var/cache/cast/installer/logs/saltstack.log
INFO[0670] results file location                         component=installer file=/var/cache/cast/installer/logs/results.yaml
WARN[0670] first failed state                            comment="One or more requisite failed: sift.scripts.vshot.sift-scripts-vshot" component=installer run_num=574 sls=sift.scripts.vshot
INFO[0670] statistics                                    component=installer failed=32 success=643 total=675
INFO[0670] salt-call completed but had failed states     component=installer
```

can you please provide the two log files referenced at the end.

**Comment by ekristen** [comment]
@moondog69 if you can provide the above we can help troubleshoot.

**Comment by ekristen** [comment]
There were some underlying issues with m2crypto that changed, this has been resolved in the latest v2024.02.24 version
